src/App.tsx

```diff
@@ -1,24 +1,221 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect,
+  useMemo,
+  useState,
+  useCallback,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoFilter } from './types/TodoFilter';
+import { ErrorNotification } from './types/ErrorNotification';
+import { createTodo, deleteTodo, getTodos } from './api/todos';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { TodoError } from './components/TodoError/TodoError';
+import { Todo } from './types/Todo';
+import { RequestTodos } from './types/RequestTodos';
+import { client } from './utils/fetchClient';
 
-const USER_ID = 0;
+const USER_ID = 10644;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoFilter, setTodoFilter] = useState<TodoFilter>(TodoFilter.ALL);
+  const [errorNotification, setErrorNotification]
+    = useState<ErrorNotification>(ErrorNotification.NONE);
+  const [search, setSearch] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [todosForDeleting, setTodosForDeleting] = useState<number[]>([]);
+  const [completedID, setCompletedsID] = useState<number[]>([]);
+
+  const initialTodos = useMemo(() => {
+    return (todos.filter(todo => {
+      switch (todoFilter) {
+        case TodoFilter.ACTIVE:
+          return !todo.completed;
+        case TodoFilter.COMPLETED:
+          return todo.completed;
+        default:
+          return todo;
+      }
+    }));
```

Comment:
No need to filter in case when TodoFilter ia All.
NOTE: `Array.filter` callback shoud return `boolean` value - in you default case you return an `object`